.github/workflows/dev.yml

```diff
@@ -25,6 +25,9 @@ jobs:
           virtualenvs-create: false
       - name: Install dependencies
         run: poetry install
+      - name: "Pre-processing for demo environment"
+        run: |
+          ./demo/preproc.sh
```

Comment:
Any reason to introduce bash and not just stick with Python?

Reply:
Not beyond "simplest tool for the job" (two cp commands)